penpost/penpost-web
# Jk/bugfix/landing query

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 

Sets the userID hook in App.js directly on Login page once retrieved from storage.
Clears apollo cache on logout (Header.js)
Passes isLoggedIn hook from App to Home (Home already written to take data)

Issues closed #58 

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

N/A

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

App: line 39 & 48
Login: line 17

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Logout, login as new user, verify that you do not see previous user data and new data loads properly.
